### ISSUE TITLE
3.2.1-rc1 checklist

### ISSUE THREAD

**Issue by jscook2345** [apptainer/singularity#3628]
Verify all fixes in PR #3607 are applied.

**Comment by jscook2345** [comment]
#3419 appears to be fixed in #3607 

```
$ env PATH=/usr/local/singularity/3.2.0/bin:$PATH singularity exec centos_centos7.sif cat /etc/os-release

WARNING: seccomp requested but not enabled, seccomp library is missing or too old
NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:7"
HOME_URL="https://www.centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"

CENTOS_MANTISBT_PROJECT="CentOS-7"
CENTOS_MANTISBT_PROJECT_VERSION="7"
REDHAT_SUPPORT_PRODUCT="centos"
REDHAT_SUPPORT_PRODUCT_VERSION="7"

$ env PATH=/usr/local/singularity/3.2.1/bin:$PATH singularity exec centos_centos7.sif cat /etc/os-release

NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:7"
HOME_URL="https://www.centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"

CENTOS_MANTISBT_PROJECT="CentOS-7"
CENTOS_MANTISBT_PROJECT_VERSION="7"
REDHAT_SUPPORT_PRODUCT="centos"
REDHAT_SUPPORT_PRODUCT_VERSION="7"
```

**Comment by jscook2345** [comment]
#3586 appears to be fixed in #3607 

```
$ sudo env PATH=/usr/local/singularity/3.2.0/bin:$PATH singularity instance start centos_centos7.sif c7
WARNING: seccomp requested but not enabled, seccomp library is missing or too old

INFO:    instance started successfully

$ sudo env PATH=/usr/local/singularity/3.2.0/bin:$PATH singularity exec instance://c7 cat /etc/os-release
FATAL:   a setuid installation is required to join this instance

$ sudo env PATH=/usr/local/singularity/3.2.0/bin:$PATH singularity instance stop c7
Stopping c7 instance of /home/vagrant/go/src/github.com/sylabs/centos_centos7.sif (PID=2102)

$ sudo env PATH=/usr/local/singularity/3.2.1/bin:$PATH singularity instance start centos_centos7.sif c7
INFO:    instance started successfully

$ sudo env PATH=/usr/local/singularity/3.2.1/bin:$PATH singularity exec instance://c7 cat /etc/os-release
NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:7"
HOME_URL="https://www.centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"

CENTOS_MANTISBT_PROJECT="CentOS-7"
CENTOS_MANTISBT_PROJECT_VERSION="7"
REDHAT_SUPPORT_PRODUCT="centos"
REDHAT_SUPPORT_PRODUCT_VERSION="7"

$ sudo env PATH=/usr/local/singularity/3.2.1/bin:$PATH singularity instance stop c7
Stopping c7 instance of /home/vagrant/go/src/github.com/sylabs/centos_centos7.sif (PID=2165)
```

**Comment by jscook2345** [comment]
#3299 appears to be fixed in #3607 

```
WARNING: seccomp requested but not enabled, seccomp library is missing or too old
total 16
-rw-r--r--.   1 root    root    12082 May 27 01:45 anaconda-post.log
lrwxrwxrwx.   1 vagrant vagrant     7 May 27 02:04 bin -> usr/bin
drwxr-xr-x.  18 root    root     3400 May 27 00:48 dev
lrwxrwxrwx.   1 vagrant vagrant    36 May 27 02:04 environment -> .singularity.d/env/90-environment.sh
drwxr-xr-x.  47 root    root     1936 May 27 01:45 etc
drwxr-xr-x.   3 vagrant vagrant    60 May 27 02:04 home
lrwxrwxrwx.   1 vagrant vagrant     7 May 27 02:04 lib -> usr/lib
lrwxrwxrwx.   1 vagrant vagrant     9 May 27 02:04 lib64 -> usr/lib64
drwxr-xr-x.   2 root    root        3 Apr 11  2018 media
drwxr-xr-x.   2 root    root        3 Apr 11  2018 mnt
drwxr-xr-x.   2 root    root        3 Apr 11  2018 opt
dr-xr-xr-x. 111 root    root        0 May 27 00:48 proc
drwxr-x---.   2 root    root      123 Mar  5 17:36 root
drwxr-xr-x.  11 root    root      157 Mar  5 17:36 run
lrwxrwxrwx.   1 vagrant vagrant     8 May 27 02:04 sbin -> usr/sbin
lrwxrwxrwx.   1 vagrant vagrant    24 May 27 02:04 singularity -> .singularity.d/runscript
drwxr-xr-x.   2 root    root        3 Apr 11  2018 srv
drwxr-xr-x   13 root    root        0 May 27 00:48 sys
drwxrwxrwt.   8 root    root     4096 May 27 01:49 tmp
drwxr-xr-x.  13 root    root      212 Mar  5 17:34 usr
drwxr-xr-x.  18 root    root      259 Mar  5 17:34 var
(END)
[5]+  Stopped                 env PATH=/usr/local/singularity/3.2.0/bin:$PATH singularity exec centos_centos7.sif ls -l / | less

$ env PATH=/usr/local/singularity/3.2.1/bin:$PATH singularity exec centos_centos7.sif ls -l / | less
total 16
-rw-r--r--.   1 root    root    12082 May 27 01:45 anaconda-post.log
lrwxrwxrwx.   1 vagrant vagrant     7 May 27 02:04 bin -> usr/bin
drwxr-xr-x.  18 root    root     3400 May 27 00:48 dev
lrwxrwxrwx.   1 vagrant vagrant    36 May 27 02:04 environment -> .singularity.d/env/90-environment.sh
drwxr-xr-x.  47 root    root     1936 May 27 01:45 etc
drwxr-xr-x.   3 vagrant vagrant    60 May 27 02:04 home
lrwxrwxrwx.   1 vagrant vagrant     7 May 27 02:04 lib -> usr/lib
lrwxrwxrwx.   1 vagrant vagrant     9 May 27 02:04 lib64 -> usr/lib64
drwxr-xr-x.   2 root    root        3 Apr 11  2018 media
drwxr-xr-x.   2 root    root        3 Apr 11  2018 mnt
drwxr-xr-x.   2 root    root        3 Apr 11  2018 opt
dr-xr-xr-x. 112 root    root        0 May 27 00:48 proc
drwxr-x---.   2 root    root      123 Mar  5 17:36 root
drwxr-xr-x.  11 root    root      157 Mar  5 17:36 run
lrwxrwxrwx.   1 vagrant vagrant     8 May 27 02:04 sbin -> usr/sbin
lrwxrwxrwx.   1 vagrant vagrant    24 May 27 02:04 singularity -> .singularity.d/runscript
drwxr-xr-x.   2 root    root        3 Apr 11  2018 srv
drwxr-xr-x   13 root    root        0 May 27 00:48 sys
drwxrwxrwt.   8 root    root     4096 May 27 01:49 tmp
drwxr-xr-x.  13 root    root      212 Mar  5 17:34 usr
drwxr-xr-x.  18 root    root      259 Mar  5 17:34 var
(END)
```

**Comment by jscook2345** [comment]
3.2.1 is out the door!
